app/domain/authentication/authn_k8s/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,1135 @@
+# Conjur Kubernetes Authentication (`authn-k8s`) Troubleshooting Guide
```

Comment:
Overall, I think this is really solid. The one thing I'd like to see is a table of contents, and I would move most of these sections up a level or two (i.e. going from ###### to ### or ####)

Reply:
Thanks. Yeah, I've got too many section levels, will fix.